test-cloud/test-cloud-ha-cep/src/test/filtered-resources/filtered.properties

```diff
@@ -0,0 +1 @@
+build.directory=${build.directory}
```

Comment:
I would prefer to pass directory for AMQ resources as system property, configured in Maven pom file. That way user can configure it according to its needs and you can get rid of extra step (filtering and loading of properties)